packages/stark-ui/src/modules/table/entities/column-properties.intf.ts

```diff
@@ -60,6 +60,11 @@ export interface StarkTableColumnProperties {
 	 */
 	label?: string;
 
+	/**
+	 * Value to be shown as the column's footer.
+	 */
+	footerValue?: any;
```

Comment:
In fact I think we should to set `string|number` to make our API clear enough... later on we might change this value to anything as @nicanac suggests.

Also, I think we should add the following to the JSDoc: `Default: if there is no footer defined here and in any other column, then it won't be displayed. Otherwise, if at least one of the other columns defines a footer, then the footer of this culumn will be displayed as empty`. What do you think?

Reply:
Yes, it was a bit poor in doc, thanks

Reply:
and also, I've a remark of tslint when I use string|number , it also requires the value undefined.
Is it ok for you

Reply:
but I think `footerValue?: string|number;` already covers the `undefined` type right? Or am I missing something?

Reply:
![image](https://user-images.githubusercontent.com/51330762/73737727-79cbc600-4743-11ea-9734-bb1c710638e2.png)
![image](https://user-images.githubusercontent.com/51330762/73737734-7d5f4d00-4743-11ea-9202-124457aaa0ac.png)

Reply:
ah ok, the getter/setter... yeah, in that case you should add `|undefined`